metrics/xrootd_metrics.go

```diff
@@ -438,6 +444,36 @@ var (
 		Help: "Number of scheduler threads",
 	}, []string{"state"})
 
+	ThreadCreations = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_creations",
+		Help: "Number of scheduler thread creations",
+	})
+
+	ThreadDestructions = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_destructions",
+		Help: "Number of scheduler thread destructions",
+	})
+
+	ThreadLimitReached = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_limit_reached",
+		Help: "Number of scheduler thread limit reached",
```

Comment:
This help message doesn't parse for me. Maybe "Number of times the scheduler thread limit has been reached"?